### ISSUE TITLE
[Bug] Mangas from some sources not showing even after download

### ISSUE THREAD

**Issue by PureFate** [tachiyomiorg/tachiyomi#3514]
### Device information
* Tachiyomi version: v0.9.2
* Android version: 7
* Device: Oppo R11

## Steps to reproduce
Not sure how to reproduce but specific manhwas from some sources not showing. I.e. Immortal Swordsman in Reverse world in Manhua Plus extension. 

### Expected behavior
Should normally display chapter once finished loading the image

### Actual behavior
Loading icon finishes but no image for the chapter shows up, just displays a blank screen

### Other details
I have tried downloading to see whether it would load but it didn't work so i looked for the downloaded chapter in my files and was able to view the image in another app.

**Comment by scb261** [comment]
Could be connected to #2614, where some pages aren't shown but not whole chapters, I'm not sure if the author meant whole chapters or only some images.

**Comment by PureFate** [comment]
> Could be connected to #2614, where some pages aren't shown but not whole chapters, I'm not sure if the author meant whole chapters or only some images.

**Comment by PureFate** [comment]
This is kinda related to #2614, but i haven't been able to fix it with some of the methods listed there.

**Comment by hXtreme** [comment]
Just wondering, did you add the manhwa to library?

**Comment by PureFate** [comment]
Yep

**Comment by scb261** [comment]
There is one more person with the bug on discord, probably problem with webtoon mode on Oppo phones

**Comment by PureFate** [comment]
Might be because my android version isn't up to date

**Comment by PureFate** [comment]
Though that is the latest version I can use without using a ROM

**Comment by scb261** [comment]
You can use vertical mode, but you need zoom each time, the person from discord also had problem with *Fit width* setting, can be different for you though. If problem is really with Oppo phones, developers probably won't fix it. Android 7 shouldn't be a problem, the app works for Android 5+

**Comment by ChilelekoC** [comment]
I have this exact bug on Tachiyomi v0.9.2 and Android 7 too where some pages in Webtoon or Continuos Vertical will load but others are completely black even when downloaded, but my phone is a Blu Vivo One Plus, and I had an even more obscure Itel S12 on hand, running the same Android and Tachiyomi versions, that was able to load the same chapters perfectly.

**Comment by scb261** [comment]
One more person with the same problem, OPPO CPH1701, android 6.0.1

**Comment by scb261** [comment]
Is there someone who had this problem? It could be fixed with new image decoder in 0.10.8, need someone to confirm

**Comment by arkon** [comment]
v0.10.8 doesn't include the new image decoder.

**Comment by PureFate** [comment]
I still have this issue with my tachiyomi which is v0.10.8, any way i can try this image decoder?

**Comment by scb261** [comment]
Sorry, my bad.

@PureFate yeah, you can still test it on preview version, it uses the new decoder https://tachiyomi.org/download/

**Comment by PureFate** [comment]
Thx i can try it rn

**Comment by arkon** [comment]
The most recent preview build doesn't include it either, since it's the exact same as the stable release. You'll have to wait for the next preview build (ETA ~6.5 hours from now).

**Comment by PureFate** [comment]
Ok np i'll test it next time i try reading something with this issur

**Comment by ChilelekoC** [comment]
I was the other person with this issue and I tried using the preview build that I downloaded today and had the same issues.
Here is the info from the app:
- App version: 0.10.5-2774 (standard, b88e444c, 51)
- Android version: 7.1.1 (SDK 25)
- Device brand: BLU
- Device manufacturer: BLU
- Device model: Vivo One Plus
- Device product name: Vivo_One_Plus

And here is a video of the issue in question (just screen recorded my phone from the desktop):

https://user-images.githubusercontent.com/69634807/107173994-2532da80-6997-11eb-812b-e5d8909bfb1f.mp4

Basically if a manhwa/webtoon/manhua has long pages, in the continuous vertical or webtoon modes, they just don't view at all and kind of locks up the reader, but switching to any other reader and the pages load just fine. And this only seems to apply specifically to long pages as the webtoon Eleceed I opened up has always worked fine in both Continuous and Webtoon possibly because Webtoon splits up pages into small and even parts.

Sorry if this is a little long winded.

**Comment by ChilelekoC** [comment]
Realized that I did not have the latest preview after looking at the github release page instead of the Tachiyomi website but after trying both builds r3085 and r3095 that had mentioned the new image decoder the app behaved exactly like the video.

**Comment by crest1993** [comment]
I think the problem is only occurs with a very long strip image even with the current preview version still the same (R3136)

**Comment by scb261** [comment]
(Following #5954) Can someone with Oppo phone and this problem confirm if it happens to all pages in webtoon mode, or only with large pages? I'm guessing this is just a case of #1908

**Comment by PureFate** [comment]
I still have the same phone in the description above and large pages haven't been loading. Recently, I was able to view them in tachiyomi by repeatedly skipping pages with the scroll bar at the bottom of the page(i.e. dragging the dot constantly till the page loaded). This method worked decently and for some long pages after I did this method would load normally but take a bit to show up.

**Comment by arkon** [comment]
Duplicate of #1908
